server/src/instant/util/int.clj

```diff
@@ -0,0 +1,11 @@
+(ns instant.util.int)
```

Comment:
nit would call this utility `number`, as I think we'll likely want other utlity functions that use longs etc